### DISCUSSION TITLE
Parse error: syntax error, unexpected '[', expecting ')' in /home2/deliv597/public_html/index.php on line 34

### DISCUSSION THREAD

**Discussion by everton082** [frontendbr/forum#2014] [Perguntas e Respostas]
Boa noite estou com este problema:  Parse error: syntax error, unexpected '[', expecting ')' in /home2/deliv597/public_html/index.php on line 34

Abaixo esta o código a linha 34
![Design sem nome (1)](https://user-images.githubusercontent.com/78574093/106977337-e9023e80-6738-11eb-9d25-a7a6b140e2cb.jpg)

**Comment by rhuangabrielsantos** [comment]
Aparentemente o código está tudo certo, você consegue mandar o código completo aqui pra discussão, não apenas um print?

Use o configuração de código utilizando as `

Exemplo abaixo:

```
<?= 'Hello World'; ?>
```

Ou criar um gist se for muito grande, assim conseguiremos ser mais assertivos.

**Comment by ninetails** [comment]
tenta trocar
* na linha 34: `[` por `array(`
* na linha 43: `]` por `)`

Se funcionar é que provavelmente teu servidor está rodando numa versão antiga do PHP.
